Sources/StreamChatUI/UIConfig.swift

```diff
@@ -9,7 +9,7 @@ public typealias UIConfig = _UIConfig<NoExtraData>
 
 public struct _UIConfig<ExtraData: ExtraDataTypes> {
     /// A view used as an online activity indicator (online/offline).
-    public var onlineIndicatorView: UIView.Type = _ChatOnlineIndicatorView<ExtraData>.self
```

Comment:
The aim is for the type to have the most generic possible. Any reason why this was changed? 🤔